hello/chart/examples/response/BATTERY.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://github.com/hello-nrfcloud/proto/transformed/PCA20035%2Bsolar/historical-data",
```

Comment:
Since we do not pass responses through the per-model transformation, use

```suggestion
  "@context": "https://github.com/hello-nrfcloud/proto/historical-data-response",
```